dao/src/main/kotlin/repositories/ortrun/DaoOrtRunRepository.kt

```diff
@@ -138,6 +140,13 @@ class DaoOrtRunRepository(private val db: Database) : OrtRunRepository {
             ListQueryResult(emptyList(), parameters, 0L)
         }
 
+    override fun listActiveRuns(): List<ActiveOrtRun> = db.blockingQuery {
+        OrtRunDao.find {
+            OrtRunsTable.status eq OrtRunStatus.CREATED or
+                    (OrtRunsTable.status eq OrtRunStatus.ACTIVE)
+        }.map { ActiveOrtRun(it.id.value, it.createdAt, it.traceId) }
```

Comment:
I think this is typically the case I would't use an ORM and run instead some kind of native SQL (https://github.com/JetBrains/Exposed/issues/118): because even is you say the opposite in the commit message, _the full ORT run is still loaded from the DB_ no ?

Reply:
In the commit message, I talk about "full `OrtRun` objects with all their relations".  This is no longer the case with this new model class; only a single `SELECT` statement is executed.

Reply:
Understood.
But you get my point ? You are still loading 16 columns from the DB when 3 are needed.

Reply:
I think this is acceptable. There is currently no precedence where we execute plain SQL statements, AFAIK.